src/client/sandbox/event/focus-blur.js

```diff
@@ -115,7 +115,7 @@ export default class FocusBlurSandbox extends SandboxBase {
             this._restoreElementNonScrollableParentsScrollState(this.scrollState.elementNonScrollableParentsScrollState);
     }
 
-    _raiseEvent (el, type, callback, withoutHandlers, isAsync, forMouseEvent, preventScrolling) {
+    _raiseEvent (el, type, callback, withoutHandlers, isAsync, forMouseEvent, preventScrolling, relEl) {
```

Comment:
Rename parameter: `relEl` -> `relatedTarget`

Reply:
in context of this issue or create separate one?

Reply:
in context

Reply:
Hammerhead already supports `relatedTarget` property for mouse and required drag events. It works for mouse events in testcafe, but does not work for drag events. So support of drag events should be fixed in testcafe repo

Reply:
I've created an issue https://github.com/DevExpress/testcafe/issues/2271 in testcafe repository